hooks/useAlternateObservationZones.ts

```diff
@@ -0,0 +1,123 @@
+import {UseQueryResult, useQuery} from '@tanstack/react-query';
+import {polygon} from '@turf/helpers';
+import axios, {AxiosError} from 'axios';
+import {safeFetch} from 'hooks/fetch';
+import {LoggerContext, LoggerProps} from 'loggerContext';
+import {useContext} from 'react';
+import {xml2json} from 'xml-js';
+interface KMLPoint {
+  coordinates: string;
+}
+
+interface KMLPlacemark {
+  name: {
+    _text: string;
+  };
+  description?: string;
+  Point?: KMLPoint;
+  LineString?: {
+    coordinates: string;
+  };
+  Polygon?: {
+    outerBoundaryIs: {
+      LinearRing: {
+        coordinates: {
+          _text: string;
+        };
+      };
+    };
+  };
+  MultiGeometry?: {
+    Polygon?: {
+      outerBoundaryIs: {
+        LinearRing: {
+          coordinates: {
+            _text: string;
+          };
+        };
+      };
+    };
+  };
+}
+
+interface KMLDocument {
+  name: string;
+  Folder: {
+    Placemark: KMLPlacemark[];
+  };
+}
+
+interface KMLData {
+  kml: {
+    Document: KMLDocument;
+  };
+}
+
+interface AlternateObservationZones {
+  geometry: GeoJSON.Polygon;
+  properties: {
+    name: string;
+  };
+}
+
+interface UseAlternateObservationZonesOptions {
+  enabled?: boolean;
+}
+
+export const useAlternateObservationZones = (url?: string, options?: UseAlternateObservationZonesOptions): UseQueryResult<AlternateObservationZones[] | null, AxiosError> => {
```

Comment:
`url === undefined` always means `options.enabled === false`. It would be a mistake to pass `useAlternateObservationZones(somethingReal, false)` - let's change the signature to `(url?: string)` and compute enabled directly as `!!url`.